### PR TITLE
Windows 2016 Developer set up deprecation

### PR DESCRIPTION
**Pull request by BRMcLaren**
This PR deprecates the 2016 installation setup components as the second half of #3905 . As windows, 2019 ~= for Windows 10 in regards to the installation of components with respect to Intel SGX, also tacked on some consolidation to simplify the script.

### CODE REVIEW

File: scripts/install-windows-prereqs.ps1
Comment (rs--) [inline scripts/install-windows-prereqs.ps1:574]: Since we ran this code path automatically when running on 2019, should this check be removed?

**Review by rs-- — APPROVED** [review]

**Comment by BRMcLaren** [comment]
@rs--  Rebased and fixed the recommendation, added as a separate commit. Thanks for pointing it out.

File: scripts/install-windows-prereqs.ps1
Comment (BRMcLaren) [reply scripts/install-windows-prereqs.ps1:574]: You could think of this check as to see if the user is setting up on ACC hardware, as opposed to a non-ACC vm as line 575 will not run on a non-SGX VM.

We never encountered this edge case before due to us always using host verification 2016 boxes (in which is actually a testing hole on our part). So the above would need to run when running a SGX1FLC setup but not a SGX1FLC-NoIntelDrivers.

I can add a comment regarding this as it's tribal info? but I am of the opinion the above implementation is correct.

Comment (rs--) [reply scripts/install-windows-prereqs.ps1:574]: Works for me

**Review by CyanDevs — APPROVED** [review]
LGTM!

**Comment by BRMcLaren** [comment]
Someone is decoupling their infra from this, they asked me to wait a day before merging so adding do not merge label for now.

**Pull request merged**
